tls_codec/derive/src/lib.rs

```diff
@@ -231,15 +231,15 @@ fn impl_tls_size(parsed_ast: TlsStruct) -> TokenStream2 {
             field_paths,
         }) => {
             quote! {
-                impl#generics tls_codec::Size for #ident#generics {
+                impl #generics tls_codec::Size for #ident #generics {
```

Comment:
@franziskuskiefer FYI, these were auto-corrected by `cargo fix --edition`:

```
error: prefix `impl` is unknown
   --> tls_codec/derive/src/lib.rs:234:17
    |
234 |                 impl#generics tls_codec::Size for #ident#generics {
    |                 ^^^^ unknown prefix
    |
    = note: prefixed identifiers and literals are reserved since Rust 2021
help: consider inserting whitespace here
    |
234 -                 impl#generics tls_codec::Size for #ident#generics {
234 +                 impl #generics tls_codec::Size for #ident#generics {
```

See: https://doc.rust-lang.org/edition-guide/rust-2021/reserving-syntax.html

Reply:
👍🏻 well, if that's the only fix needed for 2021 I'm fine with that 😄